README.md

```diff
@@ -1,2 +1,23 @@
 # HW01
-Edit the readme
+Nabilah Sammudin
+7/6/2020
+
+## Brief Biography
+
+My name is **Nabilah** and I am from **Malaysia**. I graduated from **Purdue University** back in 2017 and I am currently working as a research technician in the same lab I did my research as an undergraduate. 
```

Comment:
Hi Nabilah! It's nice to e-meet you! I also graduated in 2017 and have been working as a research tech since. It's cool to see another student with a similar background

Reply:
Hi Laura, nice to e-meet you as well! I read from your homework submission that you are an incoming Integrative Biology grad student. I think the things that you have been doing after graduation is super cool! Are you excited to start your PhD soon?